src/data/profiles.js

```diff
@@ -144,6 +144,71 @@ const officers = [
 		programming, I like gaming, playing bass, watching anime and rewatching Gilmore Girls,
 		and spending way too much time on Discord. You can always find me losing sleep from
 		procrastination.`
+	},
```

Comment:
Are they really? I thought advisors are always listed last.

Reply:
(Replying here b/c I was added as a reviewer, just saw this in my e-mails)

I tried to search through old commits and wayback machine but wasn't able to find a good example of a page *with* interns... so you can choose to do whatever feels correct. Or, feel free to retire me from the list to avoid having this issue 🤣